docs/Monitored_Only_ZFS.md

```diff
@@ -100,12 +74,12 @@ vagrant ssh <server>  and as root (sudo su ) run the following commands:
     genhostid
 ```
 
-The IML GUI should show that the LNET and NID Configuration is updated (IP Address 10.73.20.x to use `Lustre Network 0`). All alets are cleared.
+The IML GUI should show that the LNET and NID Configuration is updated (IP Address 10.73.20.x to use `Lustre Network 0`). All alerts are cleared.
 
 
 ## Creating a monitored only zfs based Lustre filesystem
 
-The lustre filesystem will be created from the command line on zpools and IML GUI will be used to scan for the filesytem.
+The lustre filesystem will be created from the command line on zpools. IML GUI will be used to scan for the filesytem.
```

Comment:
There is something grammatically wrong with "from the command line on zpools. IML GUI will be used".   I think it was just fine with the _and_ that has been removed.